src/psyclone/psyir/nodes/node.py

```diff
@@ -1506,14 +1506,22 @@ def path_from(self, ancestor):
         ''' Find the path in the psyir tree between ancestor and node and
         returns a list containing the path.
 
+        The result of this method can be used to find the node from its
+        ancestor for example by:
+        >>> index_list = node.path_from(ancestor)
```

Comment:
This isn't rendering correctly when I build the user guide. Perhaps you need a blank line before the first '>>>' or maybe you need to indent the '>>>' lines?

Reply:
Fixed and built locally and seems fine, just needed a blank line.

Reply:
All good now, thanks.